testground/README.md

```diff
@@ -80,12 +80,22 @@ To simplify cluster setup, we are introducing a stateless mode.
 You need to have the `cronosd` in `PATH`.
 
 ```bash
-$ nix run github:crypto-org-chain/cronos#stateless-testcase gen /tmp/data/out 3 7
+$ nix run github:crypto-org-chain/cronos#stateless-testcase gen /tmp/data/out \
+  --validators 3 \
```

Comment:
need change params for validators and fullnodes right?

Reply:
yes, no defaults for them, but using flag makes it easier to read.